README

```diff
@@ -73,12 +73,123 @@ from using single-DES cryptosystems.  Among these is a configuration
 variable that enables "weak" enctypes, which defaults to "false"
 beginning with krb5-1.8.
 
-Major changes in 1.14
+Major changes in 1.15
 ---------------------
 
-krb5-1.14 changes by ticket ID
+Administrator experience:
+
+* Add support to kadmin for remote extraction of current keys without
+  changing them (requires a special kadmin permission), with the
+  exception of highly protected keys.
+
+* Restore recursive dump capability for DB2 back end, so sites can
+  more easily recover from database corruption resulting from power
+  failure events.
+
+* Add DNS auto-discovery of HTTP proxy by URI records.
```

Comment:
I'd flesh this out: "Add DNS auto-discovery of KDC and kpasswd servers from URI records, in addition to SRV records.  URI records can convey TCP and UDP servers and master KDC status in a single DNS lookup, and can also point to HTTPS proxy servers."